examples/flutter_gallery/.idea/runConfigurations/main_dart.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="flutter_gallery" type="FlutterRunConfigurationType" factoryName="Flutter">
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter">
```

Comment:
Ah, you are right. These are just used for when this module is opened individually. `main.dart` sgtm, sorry for the noise.